### PR TITLE
[native] Correct the cumulative memory usage calculation

### PR DESCRIPTION
**Pull request by xiaoxmeng**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by mbasmanova — APPROVED** [review]
Thank you for the fix. It would be nice to add a test.

**Review by tanjialiang — APPROVED** [review]

**Comment by xiaoxmeng** [comment]
> Thank you for the fix. It would be nice to add a test.

We already have a test to cover that and the test has been changed accordingly.

File: presto-native-execution/presto_cpp/main/PrestoTask.cpp [hunk 1]
```diff
@@ -561,7 +561,7 @@ void PrestoTask::updateMemoryInfoLocked(
   const double sinceLastPeriodMs = currentTimeMs - lastTaskStatsUpdateMs;
 
   prestoTaskStats.cumulativeUserMemory +=
-      (averageMemoryForLastPeriod * sinceLastPeriodMs) / 1000;
+      (averageMemoryForLastPeriod * sinceLastPeriodMs);
```
Comment (majetideepak) [inline presto-native-execution/presto_cpp/main/PrestoTask.cpp:564]: @xiaoxmeng can you clarify why this change is required? Thanks.

Comment (xiaoxmeng) [reply presto-native-execution/presto_cpp/main/PrestoTask.cpp:564]: IIRC, java use ms instead of us. The change is to align with java. without this change, native reports much less accumulative memory than java.
